src/providers/aws/mod.rs

```diff
@@ -79,7 +79,7 @@ impl AwsProvider {
 
     #[cfg(not(test))]
     fn endpoint_for(key: &str, use_latest: bool) -> String {
-        const URL: &str = "http://169.254.169.254/2019-10-01";
+        const URL: &str = "http://169.254.169.254/2021-01-03";
```

Comment:
Minor: this would be nicer as a separate commit.

Reply:
Hmm, I can't seem to find any documentation about what actually changes in each version (the only section on the topic I found was https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html#instancedata-meta-data-retrieval-examples). Was it just trial-and-error that got you to figure out that this was the earliest version that had `meta-data/ipv6`?

Reply:
Those docs are [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-categories.html).

Moved the version bump to a prep commit.